app/modules/import-export/utils.php

```diff
@@ -80,4 +80,20 @@ public static function get_registered_cpt_names() {
 
 		return array_keys( $post_types );
 	}
+
+	/**
+	 * Transform a string name to title format.
+	 *
+	 * @param $name
+	 * @return string
+	 */
+	public static function transform_name_to_title( $name ): string {
+		if ( empty( $name ) ) {
+			return '';
+		}
+
+		$title = str_replace( [ '-', '_' ], ' ', $name );
+
+		return ucwords( $title );
```

Comment:
I would personally get rid of the title variable here as it doesn't really provide much extra context and used only once but up to you

Reply:
In my opinion, it makes it more readable.